tests/k8s_test.go

```diff
@@ -112,24 +112,42 @@ func checkEdgeCreation(t *testing.T, c *CheckContext, from, to *graph.Node, relT
 }
 
 /* -- test creation of single resource -- */
-func testNodeCreation(t *testing.T, setupCmds, tearDownCmds []helper.Cmd, typ, name string) {
+func testNodeCreation(t *testing.T, setupCmds, tearDownCmds []helper.Cmd, typ, name string, fields ...string) {
 	test := &Test{
 		mode:         OneShot,
 		retries:      3,
 		setupCmds:    append(tearDownCmds, setupCmds...),
 		tearDownCmds: tearDownCmds,
 		checks: []CheckFunction{func(c *CheckContext) error {
-			_, err := checkNodeCreation(t, c, typ, "Name", name)
-			return err
+			obj, err := checkNodeCreation(t, c, typ, "Name", name)
+			if err != nil {
+				return err
+			}
+
+			m := obj.Metadata()
+
+			const fieldDoesNotExist = "DoesNotExist"
```

Comment:
how a metadata Node that have been queried will reply "DoesNotExist" ?
This should not append, as the metadata (obj.Metadata()[key]) hash key should not exist

Reply:
ok - removing